sourcemash/api/items.py

```diff
@@ -109,6 +117,22 @@ def get(self):
         user_items = UserItem.query.filter_by(user=current_user, saved=True).all()
         return {'items': [marshal(user_item.item, item_fields) for user_item in user_items]}
 
+class TrendingItemListAPI(Resource):
+
+    @login_required
+    def get(self):
+        trending_items = UserItem.query.with_entities(UserItem.item_id, func.count())     \
+                                .filter(UserItem.vote)     \
+                                .filter(UserItem.last_modified > (datetime.utcnow() - timedelta(days=TRENDING_ITEMS_TIMEDELTA))) \
+                                .group_by(UserItem.item_id)        \
+                                .order_by(desc(func.count(UserItem.item_id)))      \
+                                .all()
```

Comment:
Ah missed that one. It's only the categories.py query as well, but I'll check and remove all of them if it's not necessary.